src/Scope/Subst.hs

```diff
@@ -162,7 +164,7 @@ instance Substitutable Term where
     pure TmEqs <*> substM tms
   
   freeVars (TmVarL x tp) = Map.singleton x tp
-  freeVars (TmVarG g x tis as tp) = freeVars (fsts as)
+  freeVars (TmVarG g x tgs tis as tp) = freeVars tgs <> freeVars tis <> freeVars (fsts as)
```

Comment:
This is a change in behavior; I think it's more correct, but doesn't seem to matter.